app/assets/stylesheets/components/_util.scss

```diff
@@ -1,5 +1,9 @@
 // supplemental utility classes
 
+// scss-lint:disable ImportantRule
+.display-block { display: block !important; }
```

Comment:
basecss does have a class called `block`, which provides the `display: block` style, albeit without the '!important'.

I named this class after the basecss `display-none` class, which provides both the 'important' declaration and the `display: none` style. I'm not sure if this was a good convention to follow, as it is definitely opaque in both cases (neither class mentions the 'important' part)